### ISSUE TITLE
Follow request from another account

### ISSUE THREAD

**Issue by OniriCorpe** [feditext/feditext#336]
**Is your feature request related to a problem? Please describe.**
For now, to send a follow request, we need to be using the account from where we want to send the request 

**Describe the solution you'd like**
It would be very useful if we could send a follow request when not using the right account, by long pressing the "Follow" button, as we can already do for favs and boosts

**Describe alternatives you've considered**
The only alternative is to copy the user @ then manually switch account, paste their @ in the search and finally make the follow request 
And maybe switch account again 

**Additional context**
N/A

**Comment by VyrCossont** [comment]
Good idea. I don't think the follow button has any existing long-press behavior so this shouldn't be too hard.
